account_move_batch_validate/wizard/move_marker.py

```diff
@@ -38,11 +38,16 @@ class AccountMoveMarker(orm.TransientModel):
             ('mark', 'Mark for posting'),
             ('unmark', 'Unmark for posting'),
         ], "Action", required=True),
-        'eta': fields.integer('Seconds to wait before starting the jobs')
+        'eta': fields.integer('Seconds to wait before starting the jobs'),
+        'journal_ids': fields.many2many('account.journal',
+                                        'account_post_journal_rel',
+                                        'wiz_id', 'journal_id', 'Journals',
+                                        required=True),
```

Comment:
In the official wizard validate.account.move the list of period and journals is initially empty and the user must select explicitly the one he/she wants to post.

In this batch validate module, it proposes the list of existing journals and the user must remove the one he/she does not want to validate.

I personally prefer the behaviour of the official module.